packages/cli/src/plugins/mosaicWorkflowsPlugin.ts

```diff
@@ -14,47 +16,47 @@ export interface WorkflowRequestBodyType {
   name: string;
 }
 
-function mosaicWorkflows(fastify: FastifyInstance, _options, next) {
+async function mosaicWorkflows(fastify: FastifyInstance, _options) {
+  await fastify.register(websocket);
   const { fs, core } = fastify.mosaic;
 
   /**
    * Run a workflow
    */
-  fastify.post(
-    '/workflows',
-    async (req: FastifyRequest<{ Body: WorkflowRequestBodyType }>, reply: FastifyReply) => {
-      try {
-        const { route: routeReq, name, ...restParams } = req.body;
+  fastify.get('/workflows', { websocket: true }, (connection /* SocketStream */) => {
+    connection.socket.on('message', async message => {
+      const sendMessage: SendSourceWorkflowMessage = (message, status) =>
```

Comment:
Probably want to give `sendMessage` a better name... `sendWorkflowProgressMessage` maybe.